gooddata-sdk/gooddata_sdk/catalog/workspace/model_container.py

```diff
@@ -126,7 +126,7 @@ def _valid_objects(self, ctx: ValidObjectsInputType) -> ValidObjects:
 
     def filter_by_valid_objects(
         self, valid_objects: dict[str, set[str]]
-    ) -> tuple[List[CatalogDataset], List[CatalogMetric]]:
+    ) -> tuple[list[CatalogDataset], list[CatalogMetric]]:
```

Comment:
Hm, there might be some other uses of `List` in this file, I don't see it removed from the imports.

Reply:
Indeed. Removed.